### PR TITLE
BETA: Register opiumgames.is-a.dev

### PR DESCRIPTION
**Pull request by UtfanYTl**
Added `opiumgames.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/opium.json
Comment (21Z) [inline domains/opium.json:1]: Empty JSON file

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/opiumgames.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "UtfanYTl",
+        "email": "ajihager@outlook.com"
+    },
+    "record": {
+        "CNAME": "opiumgames.netlify.app"
```
Comment (21Z) [inline domains/opiumgames.json:7]: Netlify is not supported

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/opiumgames.json (hunk 1 above)
Comment (Stef-00012) [inline domains/opiumgames.json:7]: as 21Z said, netlify is not supported

File: domains/opium.json
Comment (Stef-00012) [inline domains/opium.json:1]: as 21Z said, you can't add an empty file

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**